plugins/transforms/selectvalues/src/test/java/org/apache/hop/pipeline/transforms/selectvalues/SelectValues_LocaleHandling_Test.java

```diff
@@ -145,6 +137,6 @@ private void executeAndCheck(String locale, String expectedWeekNumber) throws Ex
 
     List<Object[]> execute = PipelineTestingUtil.execute(transform, 1, true);
```

Comment:
this is returning a formatted date, rather than the expected number of week. Probably the defaultFormatLocale is not working as expected. Heads up are more than welcome.